cypress/integration/standalone-iframe.spec.js

```diff
@@ -1,15 +1,21 @@
 import React from "react";
 import { getByCypressTag } from "../utils";
 
-// This spec does not hit firestore for the answers
+// This spec does not hit FireStore for the answers
```

Comment:
```suggestion
// This spec does not hit Firestore for the answers
```
If google is to be believed. 😉